Saturation/Splitter.hpp

```diff
@@ -91,10 +91,10 @@ _solver=0;
 private:
   friend class Splitter;
 
-  SATSolver::Status processDPConflicts();
```

Comment:
Status feels a bit too general here. How about saying `SAT::Status`` like I tried?

Reply:
Fine by me - the nice thing about `enum class` is we should get a type error if it resolves ambiguously or surprisingly.